data_model/src/transaction.rs

```diff
@@ -335,7 +335,9 @@ mod candidate {
 
     impl SignedTransactionCandidate {
         fn validate(self) -> Result<SignedTransactionV1, &'static str> {
+            #[cfg(not(target_family = "wasm"))]
```

Comment:
this is a bit hackish but I think it's ok if it's well documented

Reply:
Added comment

Reply:
this should best be documented on the module level

Reply:
even on the crate level. Somewhere where it will be visible to users that it applies to all structures deserialization